app/controllers/users_controller.rb

```diff
@@ -1,12 +1,76 @@
 class UsersController < ApplicationController
-  before_action :logged_in_user
-  def new; end
+  before_action :load_user, except: [:index, :new, :create ]
+  before_action :logged_in_user, except: [:new, :show, :create ]
+  before_action :correct_user, only: [:edit, :update]
+  before_action :admin_user, only: :destroy
+
+  def index
+    @users = User.lastest.paginate(page: params[:page] )
+  end
+
+  def new
+    @user= User.new
+  end
+
+  def show;end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      flash[:info] = I18n.t("Please check your email to activate your account.")
```

Comment:
`I18n.t("Please check your email to activate your account.")` chú tào lao. `I18n` là ri ah